docs/dapp/opl/router-protocol/pingpong-example.md

```diff
@@ -0,0 +1,351 @@
+  (current Polygon Amoy Gateway)
+- **`feePayerAddress`**: Your current account address
+  (copy from MetaMask or Remix)
+
+4. Deploy the contract on Polygon Amoy
+
+<details>
+  <summary> Remix Example </summary>
+
+![Deploy Polygon Amoy](../../images/opl/router-deploy-pingpong-amoy.png)
+</details>
+
+[Polygon Faucet]: https://faucet.polygon.technology/
+
+## Approving the Fee Payer
+
+After deploying the contracts, approve the **fee payer** on the Router Chain:
+
+1. Obtain Router test tokens from the [Router faucet].
+2. Approve the contracts on the [Router Explorer][feepayer].
+
+For detailed instructions on fee payer approval, see our [approval guide].
+
+[Router faucet]: https://faucet.routerprotocol.com/
+[feepayer]: https://testnet.routerscan.io/feePayer
+[approval guide]: /dapp/opl/router-protocol/approve
+
+## Executing PingPong
+
+Now that you've deployed the contracts and approved the fee payer, you can play
+**PingPong**. This process involves two steps:
+
+1. Obtaining the Request Metadata
+2. Executing the iPing function
+
+### Step 1: Obtaining Request Metadata
+
+Call the `getRequestMetadata()` function with the following parameters:
+
+| Argument     | Example Value | Description                                |
+| ------------ | ------------- | ------------------------------------------ |
+| destGasLimit | 300000        | Gas limit on destination chain             |
+| destGasPrice | 100000000000  | Gas price on destination chain             |
+| ackGasLimit  | 300000        | Gas limit on source chain for ack          |
+| ackGasPrice  | 100000000000  | Gas price on source chain for ack          |
+| relayerFees  | 10000000000   | Relayer fees on Router chain               |
+| ackType      | 3             | Acknowledge type                           |
+| isReadCall   | false         | Weather the call is read-only              |
+| asmAddress   | "0x"          | Address for the additional security module |
+
+<details>
+  <summary> Remix Example </summary>
+
+  ![Router getRequestMetadata](../../images/opl/router-metadata.png)
+</details>
+  
+:::info
+
+You will need the bytes answer in the next step, so copy it!
+
+:::
+
+For more information about request metadata, see the
+[Router documentation][metadata].
+
+[metadata]: https://docs.routerprotocol.com/develop/message-transfer-via-crosstalk/evm-guides/iDapp-functions/iSend#5-requestmetadata
+
+### Step 2: Executing iPing()
+
+To initiate the cross-chain message, call `iPing` with these parameters:
+
+| Argument                   | Value              | Description                                       |
+| -------------------------- | ------------------ | ------------------------------------------------- |
+| destChainId                | 23295              | Destination Chain ID (e.g. Sapphire)              |
+| destinationContractAddress | 0x\<your-contract> | Contract address on the destination chain         |
```

Comment:
Interesting. I thought escaping doesn't work this way and you need the html-like syntax for conflicting characters. Same below.

```suggestion
| destinationContractAddress | 0x&lt;your-contract&gt; | Contract address on the destination chain         |
```